src/tile-board/Tileboard.jsx

```diff
@@ -2,15 +2,27 @@ import React, { useState } from "react";
 import Tile from "./Tile";
 
 function TileBoard() {
-  const [arrayGrid] = useState(() => createArrayGrid());
+  const [size] = useState(5);
```

Comment:
Eventually I'm going to add a feature where the user can change the board size. I'm thinking the parent component handle will handle that, so it'll eventually become a prop for the TileBoard component. But since I haven't implemented anything for the parent component quite yet, I'm just keeping it as a state variable for now.